devise_invalidatable.gemspec

```diff
@@ -1,20 +1,20 @@
-$:.push File.expand_path('../lib', __FILE__)
+$LOAD_PATH.push File.expand_path('lib', __dir__)
 require 'devise_invalidatable/version'
 
 Gem::Specification.new do |s|
-  s.name = 'devise_invalidatable'
+  s.name = 'h4-devise_invalidatable'
   s.version = DeviseInvalidatable::VERSION.dup
```

Comment:
is this version the same as original or do we bump patch of some sort?

Reply:
Since I've changed the gem name (`h4-`) we could leave the version alone... but I'll bump it after merging this.